package.json

```diff
@@ -29,12 +29,13 @@
     "browserify": "browserify ./assets/js/index.js > ./public/js/bundle.js",
     "create:public": "mkdir -p ./public/js ./public/css ./public/images",
     "sass": "node-sass ./assets/scss/app.scss ./public/css/app.css --include-path ./node_modules",
-    "hof-transpile": "hof-transpiler ./apps/**/translations/src -w --shared ./node_modules/hof-template-partials/translations",
```

Comment:
If you're removing this then I'd also bump the `hof-bootstrap` version to latest as it needs *at least* v10.3.0 to work.